medusa/server/api/v2/search.py

```diff
@@ -0,0 +1,239 @@
+# coding=utf-8
+"""Request handler for statistics."""
+from __future__ import unicode_literals
+
+from collections import defaultdict
+from medusa import app
+from medusa.search.manual import collect_episodes_from_search_thread
+from medusa.tv.episode import Episode, EpisodeNumber
+from medusa.tv.series import Series, SeriesIdentifier
+from medusa.search.queue import (
+    BacklogQueueItem,
+    FailedQueueItem,
+    # ManualSearchQueueItem,
+    ForcedSearchQueueItem,
+)
+from medusa.server.api.v2.base import BaseRequestHandler
+
+from tornado.escape import json_decode
+
+
+class SearchHandler(BaseRequestHandler):
+    """Search queue request handler."""
+
+    #: resource name
+    name = 'search'
+    #: identifier
+    identifier = ('identifier', r'\w+')
+    #: path param
+    path_param = ('path_param', r'\w+')
```

Comment:
If you don't plan on using `path_param` in the future, you should set it to `None` here.
For example:
https://github.com/pymedusa/Medusa/blob/6065fdc1590c3c885bf353ccd75cc3cac945f73b/medusa/server/api/v2/internal.py#L31-L32